### PR TITLE
Ignore exec subresource in APIResponsivenessPrometheus

### PR DESCRIPTION
**Pull request by jupblb**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
See this comment: https://github.com/kubernetes/perf-tests/issues/1096#issuecomment-842099659

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:
See issue: https://github.com/kubernetes/perf-tests/issues/1096
See PR: https://github.com/kubernetes/test-infra/pull/21694

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ const (
 
 	currentAPICallMetricsVersion = "v1"
 
-	filters = `verb!="WATCH", subresource!="proxy"`
+	filters = `verb!="WATCH", subresource!="exec|proxy"`
```
Comment (mborsz) [inline clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go:48]: Shouldn't it be subresource!~"exec|proxy"?

Instead it's just a simple string comparison with "exec|proxy" string, not regexp.

Comment (marseel) [reply clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go:48]: Definitely It should be

Comment (jupblb) [reply clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go:48]: Fixed in https://github.com/kubernetes/perf-tests/pull/1814

**Pull request merged**